app/models/atmosphere/action.rb

```diff
@@ -0,0 +1,13 @@
+module Atmosphere
+  class Action < ActiveRecord::Base
+
+    belongs_to :appliance,
+               class_name: 'Atmosphere::Appliance'
+
+    has_many :action_logs,
+             dependent: :destroy,
+             class_name: 'Atmosphere::ActionLog'
+
+
+  end
+end
```

Comment:
We were talking about something like `action.info('some msg')`, `action.warn('some warn')`, `action.error('error')` methods created in action and used to log action evens.